chucks-pick3-web/angular-app/src/app/services/form-validation.service/form-validation.service.ts

```diff
@@ -56,6 +56,57 @@ export class FormValidationService {
     }
     return alphabetical;
   }
+  public isLowercase(input: string): boolean {
+    let lowercase = false;
+    const alphaChars = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'.split('');
+    const charactersArray = input.replace(/ /g, '').split('');
```

Comment:
Not sure if we want to change the input by removing any spaces before we split the string.  We should be taking the input as is.  Remove the replace method and just use the split method.

Reply:
Removed replace and left split method.